bitcoin-information/statistics-metrics.html

```diff
@@ -165,6 +165,7 @@ <h2 id="stats"><a href="#stats">Miscellaneous Metrics:</a></h2>
             <li><a href="https://howmanyconfs.com/" title="How Many Confs" target="_blank" rel="noopener">How Many Confs</a> (mining security comparison)</li>
             <li><a href="https://www.usefultulips.org/Combined_World_Page.html" title="P2P Trading" target="_blank" rel="noopener">P2P Trading Volumes</a></li>
             <li><a href="https://tsypruyan.github.io/" title="Tsypruyan" target="_blank" rel="noopener">Tsypruyan</a> - on-chain metrics</li>
+            <li><a href="https://finbold.com/bitcoin-roi" title="Bitcoin ROI" target="_blank" rel="noopener">Bitcoin ROI</a> - Compared to TOP stocks</li>
```

Comment:
I think this belongs at line 124 of `exchange-data.html` since it's a price / investment stat.

Reply:
Moved to line 123, as the anchor starts with Bitcoin.